archive.php

```diff
@@ -0,0 +1,30 @@
+<html>
+<head>
+<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="archive.css">
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+</head>
+<body>
+	<?php
+	include 'navbar.php';
+	?>
+	<div id="container">
+		<script>
+		var youtube_videos = ['15tLVw0V-bM','15tLVw0V-bM', '15tLVw0V-bM', '15tLVw0V-bM', '15tLVw0V-bM', '15tLVw0V-bM'
```

Comment:
Videos IDs added from the youtube channel.

On Tue, Sep 1, 2015 at 1:56 AM, Siddharth Kannan <notifications@github.com>
wrote:

> In archive.php
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54#discussion_r38397975>
> :
>
> > @@ -0,0 +1,30 @@
> > +<html>
> > +<head>
> > +<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
> > +<link rel="stylesheet" href="archive.css">
> > +<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
> > +</head>
> > +<body>
> > +	<?php
> > +	include 'navbar.php';
> > +	?>
> > +	<div id="container">
> > +		<script>
> > +		var youtube_videos = ['15tLVw0V-bM','15tLVw0V-bM', '15tLVw0V-bM', '15tLVw0V-bM', '15tLVw0V-bM', '15tLVw0V-bM'
>
> @hitesh4 <https://github.com/hitesh4>
> Add the video IDs of the videos that you find on our YouTube channel
> <https://www.youtube.com/channel/UCtnSs1CT59Rd8L6VlOAxzRA>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/students-alumni-cell-iitkgp/aam-2016/pull/54/files#r38397975>
> .
>